includes/person-functions.php

```diff
@@ -8,15 +8,15 @@
  * @param $post object | Person post object
  * @return Mixed | thumbnail HTML or void
  **/
-function get_person_thumbnail( $post ) {
+function get_person_thumbnail( $post, $rounded=true ) {
```

Comment:
Add `@param` definition for $rounded. Also, should we consider making this more generic in just taking a class string, that way we have more flexibility in the future if it needs to be displayed differently?